staticfiles/templates/contributors/home.html

```diff
@@ -154,97 +154,99 @@ <h1 class="page-header">
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/vaidehi">
               <img class="thumbnail img-responsive" src="/static/images/contributors/vaidehi.jpg" />
-              <span>Vaidehi Thete</span> 
+              <span>Vaidehi Thete</span>
             </a>
           </div>
-          
+
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/jason">
               <img class="thumbnail img-responsive" src="/static/images/contributors/jason.jpg" />
-              <span>Jason Ou</span> 
+              <span>Jason Ou</span>
             </a>
           </div>
-          
+
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/lucamatsumoto">
               <img class="thumbnail img-responsive" src="/static/images/contributors/lucamatsumoto.jpg" />
-              <span>Luca Matsumoto</span> 
+              <span>Luca Matsumoto</span>
             </a>
           </div>
-    
+
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/arcolife">
               <img class="thumbnail img-responsive" src="/static/images/contributors/arcolife.png" />
-              <span>Archit Sharma</span> 
+              <span>Archit Sharma</span>
             </a>
           </div>
-      
+
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/AditiNath">
               <img class="thumbnail img-responsive" src="/static/images/contributors/AditiNath.jpg" />
-              <span>Aditi Nath</span> 
+              <span>Aditi Nath</span>
             </a>
           </div>
 
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/ucerron">
               <img class="thumbnail img-responsive" src="/static/images/contributors/ucerron.jpg" />
-              <span>Uwe Cerron</span> 
+              <span>Uwe Cerron</span>
             </a>
           </div>
 
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/KajalGupta">
               <img class="thumbnail img-responsive" src="/static/images/contributors/KajalGupta.jpg" />
-              <span>Kajal Gupta</span> 
+              <span>Kajal Gupta</span>
             </a>
           </div>
 
-      
+
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/AmitRakesh">
               <img class="thumbnail img-responsive" src="/static/images/contributors/AmitRakesh.jpg" />
-              <span>Amit Rakesh</span> 
+              <span>Amit Rakesh</span>
             </a>
           </div>
+<<<<<<< HEAD
```

Comment:
you would make changes on your branch and push to it, the PR will update automatically